MapboxVisionAR/ARRenderer.swift

```diff
@@ -372,8 +403,9 @@ class ARRenderer: NSObject, MTKViewDelegate {
 
         if let frame = dataProvider.getCurrentFrame(), let texture = makeTexture(from: frame) {
             commandEncoder.setRenderPipelineState(renderPipelineBackground)
+            commandEncoder.setVertexBytes(&textureMappingVertices, length: MemoryLayout<TextureMappingVertexIn>.size * textureMappingVertices.count, index: 0)
             commandEncoder.setFragmentTexture(texture, index: 0)
-            commandEncoder.drawPrimitives(type: .triangleStrip, vertexStart: 0, vertexCount: 4, instanceCount: 1)
+            commandEncoder.drawPrimitives(type: .triangle, vertexStart: 0, vertexCount: 6, instanceCount: 2)
```

Comment:
it should be work (https://developer.apple.com/documentation/metal/mtlrendercommandencoder/1516326-drawprimitives?language=objc):
```suggestion
            commandEncoder.drawPrimitives(type: .triangle, vertexStart: 0, vertexCount: 6)
```
```suggestion
            commandEncoder.drawPrimitives(type: .triangle, vertexStart: 0, vertexCount: 6, instanceCount: 2)
```